tests/cstor/gtest/test_uzfs.cc

```diff
@@ -3366,7 +3366,7 @@ TEST(MgmtThreadTest, RebuildSuccessSingleReplica) {
 
 	uint64_t quorum = 0;
 	EXPECT_EQ(0, dsl_prop_set_int(zinfo->main_zv->zv_name,
```

Comment:
let's use `uzfs_zinfo_set_quorum` here.

Reply:
inside `uzfs_zinfo_set_quorum` we hardcoded `quorum` value as 1 (Not allowing users to change quorum from 1 to 0)

Reply:
yes. It should use value provided by caller. Lets fix that in new PR.